codacy/values.yaml

```diff
@@ -252,8 +252,9 @@ global:
   metrics:
     serviceMonitor:
       enabled: false
-    grafana_dashboards:
+    grafana:
```

Comment:
didn't this change to global?

Reply:
but this is the global block. I am confused. :)

Reply:
sorry .. i'm drunk🤦‍♂️